config/locales/en.yml

```diff
@@ -767,7 +767,7 @@ en:
       ManageIQ::Providers::Kubernetes::ContainerManager:                    Container Provider (Kubernetes)
       ManageIQ::Providers::Kubernetes::ContainerManager::ContainerGroup:    Pod (Kubernetes)
       ManageIQ::Providers::Openshift::ContainerManager:                     Container Provider (OpenShift)
-      ManageIQ::Providers::OpenshiftEnterprise::ContainerManager:           Container Provider (OpenShift Enterprise)
+      ManageIQ::Providers::OpenshiftEnterprise::ContainerManager:           Container Provider (OpenShift Container Platform)
```

Comment:
@zgalor since we're at it I noticed that the one above is just "Container Provider (OpenShift)" but it should be "Container Provider (OpenShift Origin)". Can you fix that as well?